DOC.md

```diff
@@ -69,4 +69,13 @@ e.g
 But [other coverage tools](https://vladfilippov.com/blog/rust-code-coverage-tools/) might work as well
 
 
+## Adding test measures
 
+Optionally SonarQube can also display tests measures.
+
+This Community Rust plugin doesn't run your tests or generate tests reports for you. That has to be done before analysis and provided in the form of reports.
+
+Currently, only `junit report` formats are supported :
+
+Insert a parameter `sonar.test.reportPath` into you `sonar-project.properties` file. As an example, one of such tool 
```

Comment:
the parameter name is too generic in case of multi languages project.
I suggest to change `sonar.test.reportPath` into `sonar.rust.test.reportPath` to make it specific to this plugin

Reply:
Change made in 22e6af84a8ac085ecbaa9648e71c83dc9ce0a10a